hub/publish_test.go

```diff
@@ -145,21 +146,22 @@ func TestPublishNotAuthorizedTarget(t *testing.T) {
 func TestPublishOK(t *testing.T) {
 	hub := createDummy()
 
+	pipe, err := hub.stream.CreatePipe("")
+	assert.Nil(t, err)
```

Comment:
assert report an error and continue the execution.
require report an error and stop the test.

I use "require" to avoid null pointer panic when the following assertions expect a well known object with methods and properties.

Here, I require an `pipe` because following code is calling `pipe.write()`, but just want to assert that err is not nil. It's fine if the test continue to report other errors.

Reply:
Ok makes sense.